dataproperty/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "dataproperty" %}
```

Comment:
I apologize, but I just realized that I merged a bunch of your PRs, but only when I got to this one did I realize that the location of the files were incorrect, so nothing was being built and the tests were spuriously passing.

This file should be in `recipes/dataproperty/meta.yaml`

@conda-forge/staged-recipes -- How best to back out the incorrect merges. They were for:

- msgfy
- pathvalidate
- releasecmd
- tabledata
- tcolorpy
- typepy